FrontEnd/src/components/SignUp/components/signup/signup-form/SignUpFormContent.js

```diff
@@ -1,7 +1,25 @@
+import { useState } from "react";
 import { useForm } from "react-hook-form";
+import EyeInvisible from "../../../../authorization/EyeInvisible";
+import EyeVisible from "../../../../authorization/EyeVisible";
 import styles from "./SignUpFormContent.module.css";
 
 export function SignUpFormContentComponent() {
+  const [typePassword, setTypePassword] = useState("password")
+  const [typeConfirmPassword, setTypeConfirmPassword] = useState("password")
+
+  const passwordVisisbilityHandler = () => {
```

Comment:
And then easily toggle the value `setShowPassword(! showPassword)`

Reply:
Changed.